### DISCUSSION TITLE
Problems with merging Unity project.

### DISCUSSION THREAD

**Discussion by KarolZygmunt** [community/community#37877] [Other Feature Feedback, Questions, & Ideas]
Hello, so recently me and my friend started an unity project (3D game). We created our repo and merged the code with terrain and assets. When I opened this merged project with my unity everything worked fine beside of fbx files that were not overlaid on the map automatically. We have installed GIT LFS. If you have any question please ask me below. Best regards.

**Comment by ossanna16** [comment]
Hi there @KarolZygmunt and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by Owlrazum** [comment]
Is this still relevant?

If yes, after installation of git lfs you need to track files. If not errors were shown, the fbx files are in the repo, the merge was successful and without errors, with all conflicts resolved, I cannot tell the solution to you problem.
